pkg/logs/tail.go

```diff
@@ -201,6 +201,13 @@ func (c *TailCommand) tail(out io.Writer) {
 		// Check that our request was successful. If the server is
 		// having trouble, retry after waiting for some time.
 		if resp.StatusCode != http.StatusOK {
+			// If the response was a 404, the from time was
+			// not valid, give them an error stating this and exit.
+			if resp.StatusCode == http.StatusNotFound &&
+				c.cfg.from != 0 {
+				text.Error(out, "specified from time %d not found, either too far in the past or future", c.cfg.from)
```

Comment:
I personally prefer calling out words that _mean_ something in the tech sense, so `from` referring to a value provided by the user. Without some kind of indicator around the word (in this case I've used single quotes) I feel like it's too easy to miss what's being suggested...

```suggestion
				text.Error(out, "specified 'from' time %d not found, either too far in the past or future", c.cfg.from)
```